### PR TITLE
Instruction 1.7

### PR DESCRIPTION
**Pull request by afestein**
Generates a json file containing the tweets for each keyword.

### CODE REVIEW

**Comment by afestein** [comment]
Closes #6

File: group2_assignment2_solution.ipynb [hunk 1]
```diff
@@ -495,7 +507,10 @@
     "you have crawled for each keyword, save them into a file named w.json where w is the keyword.\n",
```
Comment (afestein) [inline group2_assignment2_solution.ipynb:507]: Note that these instructions don't quite match the file names specified in the PDF, which contain our group name.

File: group2_assignment2_solution.ipynb [hunk 2]
```diff
@@ -495,7 +507,10 @@
...
     "'''\n",
     "# saving the tweets in three json files, one for each keyword\n",
-    "#INSERT YOUR CODE HERE"
+    "#INSERT YOUR CODE HERE\n",
+    "save_to_json(k1_tweets, 'group2_computer.json')\n",
+    "save_to_json(k2_tweets, 'group2_animal.json')\n",
+    "save_to_json(k3_tweets, 'group2_movie.json')"
```
Comment (afestein) [inline group2_assignment2_solution.ipynb:513]: I really wanted to use a loop here to reduce repetition, but we are stuck with the variables `k1_tweets` etc. which are annoyingly separated from the data. The name and retrieved values should all probably be stored in single object. I am probably overthinking it:

```
for index, keyword in enumerate(keywords):
    save_to_json(data_goes_here, "group2_{keyword}.json")
```

File: group2_assignment2_solution.ipynb (hunk 1 above)
Comment (jjjjesse) [reply group2_assignment2_solution.ipynb:507]: Yeah odd. Might be worth an email?

File: group2_assignment2_solution.ipynb (hunk 2 above)
Comment (jjjjesse) [reply group2_assignment2_solution.ipynb:513]: does something like
`for index, keyword in enumerate(keywords):
    save_to_json(keyword{i + 1}_tweets, "group2_{keyword}.json")`

work?

Comment (jjjjesse) [reply group2_assignment2_solution.ipynb:513]: Although maybe not. Maybe it's better to do it the simple way. Looking at the rest of the document it seems to encourage more of that style.

**Review by jjjjesse — APPROVED** [review]
Merging, but we'll keep discussing alternate solutions.

**Comment by afestein** [comment]
Simple is fine I think. 👍

**Pull request merged**